packages/build-info/src/frameworks/remix.ts

```diff
@@ -1,24 +1,75 @@
-import { BaseFramework, Category, Framework } from './framework.js'
+import { BaseFramework, Category, DetectedFramework, Framework } from './framework.js'
+
+const CLASSIC_COMPILER_CONFIG_FILES = ['remix.config.js']
+const CLASSIC_COMPILER_DEV = {
+  command: 'remix watch',
+}
+const CLASSIC_COMPILER_BUILD = {
+  command: 'remix build',
+  directory: 'public',
+}
+
+const VITE_CONFIG_FILES = [
+  'vite.config.js',
+  'vite.config.mjs',
+  'vite.config.cjs',
+  'vite.config.ts',
+  'vite.config.mts',
+  'vite.config.cts',
+]
+const VITE_DEV = {
+  command: 'remix vite:dev',
+  port: 5173,
+}
+const VITE_BUILD = {
+  command: 'remix vite:build',
+  directory: 'build/client',
+}
 
 export class Remix extends BaseFramework implements Framework {
   readonly id = 'remix'
   name = 'Remix'
-  npmDependencies = ['remix', '@remix-run/netlify', '@remix-run/netlify-edge']
-  configFiles = ['remix.config.js']
+  npmDependencies = [
+    '@remix-run/react',
+    '@remix-run/dev',
+    '@remix-run/server-runtime',
+    '@netlify/remix-adapter',
+    '@netlify/remix-edge-adapter',
+    '@netlify/remix-runtime',
+    // Deprecated package name (deprecated in 1.6, removed in 2.0)
+    'remix',
+    // Deprecated Netlify packages
+    '@remix-run/netlify',
+    '@remix-run/netlify-edge',
+  ]
```

Comment:
maybe consider adding
```suggestion
  ]
  excludedNpmDependencies = ['@shopify/hydrogen']
```
As additional rail guard against hydrogen sites being detected as remix ones in same way we add exclusions to `Vite` detection?

Reply:
Oh, of course. I think I can remove the weird `Accury.Forced` thing with this change. 🤦🏼